docs/applications/development/clojure-deployment-with-immutant-and-wildfly-on-ubuntu-14-04.md

```diff
@@ -93,20 +93,20 @@ This guide will show how to deploy Clojure application to WildFly - widely used
 
 ## Create Sample Application
 
-Now you will create sample Clojure web application based on *Luminus* framework.
+Now, you will create a sample Clojure web application based on *Luminus* framework.
 
-1.  In your home directory run
+1.  In your home directory run:
 
 		lein new luminus clj-app
 
-	This will create new web application using Luminus framework application template. It will use `immutant` as a web server, which is the default option.
+	This will create new web application using the Luminus framework application template. It will use `immutant` as a web server, which is the default option.
 
-2.  To check everything went smoothly, run newly created application in development mode:
+2.  To check that everything went smoothly, run the newly created application in development mode:
 
 		cd clj-app
 		lein run
 
-	Now open http://&lt;public_linode_ip&gt;:3000/ in your browser and you will see the sample application main page.
+	Now, open http://&lt;public_linode_ip&gt;:3000/ in your browser, and you will see the sample application main page.
```

Comment:
maybe it will be better to use phrase like 

Open `http://youripaddress:3000` using your browser

like in [this guide](https://www.linode.com/docs/applications/development/install-gogs-on-debian-8-jessie#install-gogs) ?